bin/yaml/cpp.yaml

```diff
@@ -184,19 +184,28 @@ compilers:
               - gcc-arm-none-eabi-5_4-2016q3 # name unused in this case
           - type: tarballs
             frc:
-              check_exe: roborio/bin/arm-frc{year}-linux-gnueabi-g++ --version
-              compression: gz
-              dir: arm/frc{year}-{name}
-              untar_dir: frc{year}
-              url: https://github.com/wpilibsuite/toolchain-builder/releases/download/v{year}-{build}/FRC-{year}-Linux-Toolchain-{name}.tar.gz
-              targets:
-                - year: 2019
-                  build: 3
-                  name: 6.3.0
-                - year: 2020
-                  build: 1
-                  name: 7.3.0
-                  url: https://github.com/wpilibsuite/roborio-toolchain/releases/download/v{year}-{build}/FRC-{year}-Linux-Toolchain-{name}.tar.gz
+              - check_exe: roborio/bin/arm-frc{year}-linux-gnueabi-g++ --version
```

Comment:
A lot of this stuff seems identical in both versions - I'll merge and remove the redundancy. Thanks

Reply:
Annnd it's not actually that redundant, I just can't see :D